src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
```

Comment:
It's a good practice to extract repeated values like `16px` for `height` and `width` or `4px` for `margin-right` into SCSS variables for easier maintenance and to ensure consistency. Consider using variables defined in `variables.scss`.